.github/workflows/coveralls.yml

```diff
@@ -64,7 +64,13 @@ jobs:
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4} TESTS='-test_external_krb5'
     - name: generate coverage info
-      run: lcov -d . -c -o ./lcov.info
+      run: lcov -d . -c \
+             --exclude "${PWD}/test/*" \
+             --exclude "${PWD}/test/helpers/*" \
+             --exclude "${PWD}/test/testutil/*" \
+             --exclude "${PWD}/fuzz/*" \
+             --exclude "/usr/include/*" \
```

Comment:
`/usr/include` surprised me a bit...  but yeahok

Reply:
> `/usr/include` surprised me a bit... but yeahok

You can check what we have included here https://coveralls.io/github/openssl/openssl

Reply:
Yup, gotcha.  I'm not protesting, just expressing my surprise